src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -45,14 +49,30 @@ const GordonSchedulePanel = ({ profile, myProf }: Props) => {
       setSelectedSchedule(defaultSchedule);
 
       setLoading(false);
+
+      const getIsScheduleOpen = localStorage.getItem('isScheduleOpen');
+      if (getIsScheduleOpen !== null) {
+        setIsScheduleOpen(getIsScheduleOpen === 'true');
+      }
```

Comment:
You shouldn't need this code, since the default value of `isScheduleOpen` is whatever was stored in local storage.